### PR TITLE
Updated Privacy Policy

### PR DESCRIPTION
**Pull request by johnbaku**
Proposed updates to our Privacy Policy.

### CODE REVIEW

File: legalese/privacy_policy/en.md [hunk 1]
```diff
@@ -1,102 +1,89 @@
...
+## How do you handle browser "Do Not Track" settings?
 
-## Steps We Take To Protect Your Information
+"Do Not Track" is a privacy preference you can set in your browser if you do not want online services to collect and share certain kinds of information about your online activity.
 
-### We Encrypt Your Password
+We do not track your online browsing activity on other sites, and we do not permit third-party services to track your activity on FetLife.
 
-Believe it or not, a lot of websites still store passwords in plain text so anyone who has access to the database can see your password. We encrypt your password so we have no way to tell what your password is. *Fun tip: To see if any specific site encrypts your password request your login information through their login form and if they send you back your password then you know they don't encrypt it.*
+Because we are not in the business of selling our friends (members) out, we don't believe we need to respond differently to an individual browser's Do Not Track setting.
```
Comment (rakvat) [inline legalese/privacy_policy/en.md]: "Do Not Track" (in ") is easier to read

File: legalese/privacy_policy/en.md [hunk 2]
```diff
@@ -1,102 +1,89 @@
...
 
-### We Use SSL to Login to FetLife
+## How can I manage or delete information about me?
 
-When you login to FetLife it is done over SSL so if someone wanted to try to steal your login credentials when you login they wouldn't be able to sniff it over the network. SSL is the same technology your bank uses when you login to your account on their website.
+You can view and update your personal info at any time from your account's Edit Profile and Settings screens.
 
-### We Protect You From XSS Vulnerabilities
+You can deactivate or delete your account at any time by logging into FetLife and visiting this page.
```
Comment (rakvat) [inline legalese/privacy_policy/en.md]: should "this page" be a link?

Comment (johnbaku) [inline legalese/privacy_policy/en.md]: FIXED. Thanks.

File: legalese/privacy_policy/en.md [hunk 3]
```diff
@@ -1,102 +1,89 @@
...
-## How You Can View and Modify Your Personal Information
+Backups and web server logs are permanently deleted within 30 days of their creation.
 
-Once you login to FetLife you can view and edit your profile and personal information we have on file for you. It's easy cheesy.
+## How do you handle browser "Do Not Track" settings?
 
-## Steps We Take To Protect Your Information
+"Do Not Track" is a privacy preference you can set in your browser if you do not want online services to collect and share certain kinds of information about your online activity.
 
-### We Encrypt Your Password
```
Comment (johnbaku) [inline legalese/privacy_policy/en.md]: FIXED!

File: legalese/privacy_policy/en.md (hunk 1 above)
Comment (johnbaku) [inline legalese/privacy_policy/en.md]: FIXED!

**Pull request merged**